communications/cloud.md

```diff
@@ -1,73 +1,91 @@
-Some applications may not seem like cloud services, but use the cloud for sync and storage. For example, Apple Music is a cloud service, even though it’s only accessed through iTunes and not a web browser.
+As a distributed company working with many clients, cloud services and Software as a Service (SaaS) applications make our business possible. However, the use of insecure services can introduce unintended risks to you, your coworkers, and the company. This document outlines our policies and guidelines for evaluating and using SaaS and cloud services, balancing security with productivity.
 
-## What needs to be done to introduce a new cloud service at Lullabot?
+## What is a Cloud Service or SaaS?
 
-Read the following, and then fill out [the service request form](https://forms.gle/aPZnrCoEhfhB67ck8).
+We consider a cloud service or SaaS to be any software application hosted by the vendor and accessed over the internet. Examples include Google Suite, Dropbox, Noko, and even applications like Apple Music that use cloud sync and storage.
 
-If a new cloud service is purchased individually on PEX, and it does not require integration or access to Lullabot data stored in other services, no specific approval is required. Examples of services like this include:
+## Key Terms to Understand
 
-- Evernote
-- Arq Cloud Backup
-- Microsoft Office 365
+1. **Data Handling Policies**: Documented practices for collecting, storing, processing, and disposing of user data.
+2. **Data Usage**: How the service utilizes collected data, including AI model training and third-party sharing.
+3. **Acceptable Permissions**: Access rights a service reasonably needs to function effectively without compromising privacy or security.
+4. **Sensitive Content**: Information that could potentially harm individuals or organizations if mishandled, including PII, financial data, unpublished content, and intellectual property.
 
-Be wary of applications that use your Lullabot account to access key personal data like email, especially if they are free or are low cost.
+## Evaluation Process for New Cloud/SaaS Services
 
-Apps that integrate with services used by the Lullabot team or interact with Lullabot customers should be discussed with the relevant directors and also need review by the security committee. Since these services are used by multiple team members, and not individuals, they typically are charged to a Lullabot expense card or are invoiced. Examples of services like this include:
+### Key Questions to Ask
 
-- Geekbot, as it requires integration with our Slack instance.
-- Drift, as it interacts with Lullabot customers and gathers their personal information.
-- CircleCI, as it has access to customer owned source code.
+1. **Company Location**:
+   - Is the company based in the US or an allied country?
+   - Be cautious of companies based in countries like China, Syria, Saudi Arabia, or Russia.
```

Comment:
🤔 I wonder if we need this second line. Like some of these may not be "be cautious" but "do not use", but we don't actually have a documented list that way.

Reply:
What do you think it would take to get this list? Or should we just remove the line?